extensions/binding/org.eclipse.smarthome.binding.digitalstrom/ESH-INF/i18n/digitalstrom_de.properties

```diff
@@ -0,0 +1,128 @@
+
+DSS_BRIDGE_PARM_CERT_PARTH_LABEL = Pfad zum SLL Zertifikat
+DSS_BRIDGE_PARM_CERT_PARTH_DESC = Hier k�nnen sie den Pfad zum SLL-Zertifikat f�r den digitalSTROM-Server angeben. Sie k�nnen es vom digitalSTROM-Server herunterladen. Andernfalls wird das SSL-Zertifikat ignoriert.
+
+#ds-device
+#group-parms
+DS_DEVICE_PARM_GROUP_INFO_LABEL = Ger�te Informationen
+DS_DEVICE_PARM_GROUP_INFO_DESC = Zeigt Informationen zum digitalSTROM-Ger�t.
+
+DS_DEVICE_PARM_GROUP_SENSOR_LABEL = Sensor Setup
+DS_DEVICE_PARM_GROUP_SENSOR_DESC = Hier k�nnen Sie Priorit�ten zur Aktualliesirung der Ger�te-Sensorwerte einstellen.
+  
+#parms
+DS_DEVICE_PARM_DEV_NAME_LABEL = Device Name
+DS_DEVICE_PARM_DEV_NAME_DESC = Der Name des digitalSTROM-Ger�ts.
+
+DS_DEVICE_PARM_DSUID_LABEL = Device UID
+DS_DEVICE_PARM_DSUID_DESC = Die eindeutige Kennung des digitalSTROM-Ger�ts mit virtuellen Ger�ten.
+
+DS_DEVICE_PARM_DSID_LABEL = Device ID
+DS_DEVICE_PARM_DSID_DESC = Die eindeutige Kennung des digitalSTROM-Ger�ts.
+
+DS_DEVICE_PARM_METER_DSID_LABEL = Meter dSID
+DS_DEVICE_PARM_METER_DSID_DESC = Die eindeutige Kennung des digitalSTROM-Meters an dem das Ger�t angeschlossen ist.
+
+DS_DEVICE_PARM_DEV_HW_INFO_LABEL = Device Hardware Type
+DS_DEVICE_PARM_DEV_HW_INFO_DESC = Der Hardware Type dieses digitalSTROM-Ger�ts.
+
+DS_DEVICE_PARM_ZONE_ID_LABEL = Zonen ID
+DS_DEVICE_PARM_ZONE_ID_DESC = Die Kennung der Zone in dem dieses digitalSTROM-Ger�t eingeordnet ist.
+
+DS_DEVICE_PARM_GROUP_ID_LABEL = Gruppen ID's
+DS_DEVICE_PARM_GROUP_ID_DESC = Die Kennung der Gruppen in dem dieses digitalSTROM-Ger�t eingeordnet ist. Benutzerdefinierte sowie Funktionale-Farb-Gruppen.
+
+DS_DEVICE_PARM_OUTPUTMODE_LABEL = Ger�te Ausgangs-Modus
+DS_DEVICE_PARM_OUTPUTMODE_DESC = Zeigt dem aktuellen Ausgangs-Modus des digitalSTROM-Ger�ts z.B. 22 f�r dimmbar.
+
+DS_DEVICE_PARM_FUNC_COLOR_GROUP_LABEL = Funktionale-Farb-Gruppen
+DS_DEVICE_PARM_FUNC_COLOR_GROUP_DESC = Zeigt die Funktionale-Farb-Gruppen in welcher das digitalSTROM-Ger�t aktuell eingeornet ist z.B. yellow = Licht.
+
+
+DS_DEVICE_PARM_ACTIVE_POWER_LABEL = Wirkleistungs Aktuellisierungspriorit�t
+DS_DEVICE_PARM_ACTIVE_POWER_DESC = Setzt die Priorit�t zur Aktuallisierung des Wirkleistungs-Werts.
+
+DS_DEVICE_PARM_ELECTRIC_METER_LABEL = Stromz�hler Aktuellisierungspriorit�t
+DS_DEVICE_PARM_ELECTRIC_METER_DESC = Setzt die Priorit�t zur Aktuallisierung des Stromz�hler-Werts.
+
+DS_DEVICE_PARM_OUTPUT_CURRENT_LABEL = Ausgangsstrom Aktuellisierungspriorit�t
+DS_DEVICE_PARM_OUTPUT_CURRENT_DESC = Setzt die Priorit�t zur Aktuallisierung des Ausgangsstrom-Werts.
+
+DS_SENSOR_OPTION1 = niemals
+DS_SENSOR_OPTION2 = niedrige Priorit�t
+DS_SENSOR_OPTION3 = mittlere Priorit�t
+DS_SENSOR_OPTION4 = hohe Priorit�t
+
+#scene
+DS_APP_SCENE_LABEL = digitalSTROM Appartment-Szene/Stimmung
+DS_APP_SCENE_DESC = Dises Thing repr�sentiert eine digitalSTROM Appartment-Szene/Stimmung.
```

Comment:
typo: "Dieses"
You should not speak of "Things" here - simply say "Dies repräsentiert..."